classes/submission/form/PKPSubmissionSubmitStep1Form.php

```diff
@@ -323,7 +323,7 @@ public function setCommentsToEditor($submissionId, $commentsToEditor, $userId, $
         } else {
             $queryId = $query->getId();
             $notes = $noteDao->getByAssoc(ASSOC_TYPE_QUERY, $queryId);
-            if ($note = $notes->next()) {
+            foreach ($notes as $note) {
```

Comment:
I think this should update maximum 1 note.

Reply:
Changed